README.md

```diff
@@ -1,137 +1,139 @@
+<div align="center">
 
-## About
+# [![Eluna](src/LuaEngine/docs/Eluna.png)](https://github.com/ElunaLuaEngine/Eluna)
 
-Eluna Lua Engine &copy; is a lua engine embedded to World of Warcraft emulators. Eluna supports MaNGOS, CMaNGOS, TrinityCore and AzerothCore.
-We are currently working hard to make Eluna better from inside and outside.  
+*Unleash the power of Lua scripting in your AzerothCore server*
 
-If you are having trouble with installation or scripts, please feel free to open an issue.
-For documentation and reference see [Eluna API (AC version)](https://www.azerothcore.org/pages/eluna/index.html) and [Lua reference manual](http://www.lua.org/manual/5.2/).
+[![Discord](https://img.shields.io/badge/Discord-Join%20Us-7289DA?style=for-the-badge&logo=discord&logoColor=white)](https://discord.com/invite/ZKSVREE7)
+[![Lua](https://img.shields.io/badge/Lua-5.2-2C2D72?style=for-the-badge&logo=lua&logoColor=white)](http://www.lua.org/manual/5.2/)
+[![AzerothCore](https://img.shields.io/badge/AzerothCore-Integrated-darkgreen?style=for-the-badge)](http://www.azerothcore.org/)
 
+---
+</div>
 
-## Community
+> [!NOTE]
+> mod-eluna © is a powerful Lua scripting engine embedded into the AzerothCore emulator. We are committed to continuously improving mod-eluna for both developers and server administrators.
 
-You can join the official Eluna Discord server, where you'll be able to find resources, releases and support provided by the community:
-<a href="https://discord.gg/bjkCVWqqfX">
-    <img src="https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat" />
-</a>
+<details>
+<summary>🎯 Table of Contents</summary>
 
-The official Azerothcore Discord server offers a channel dedicated to lua development as well:
-<a href="https://discord.gg/gkt4y2x">
-    <img src="https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat" />
-</a>
+- [Acknowledgements](#-acknowledgements)
+- [Support & Resources](#-support--resources)
+- [Installation](#-installation)
+- [Documentation](#-eluna-documentation)
+- [Useful Links](#-useful-links)
+- [Community Additions](#%EF%B8%8F-community-additions)
+- [Contributing](#-contributing)
+</details>
 
-# ![logo](https://raw.githubusercontent.com/azerothcore/azerothcore.github.io/master/images/logo-github.png) mod-eluna for AzerothCore
-- Latest build status with azerothcore: [![Build Status](https://github.com/azerothcore/mod-eluna/workflows/core-build/badge.svg?branch=master&event=push)](https://github.com/azerothcore/mod-eluna)
+## 🌟 Acknowledgements
+<div align="center">
 
-[english](README.md) | [中文说明](README_CN.md) | [Español](README_ES.md)
+***Built on the shoulders of giants***
+</div>
 
-An [Eluna](https://github.com/ElunaLuaEngine/Eluna) module for AzerothCore.
+mod-eluna is a fork of the original [Eluna](https://github.com/ElunaLuaEngine/Eluna) project. We extend our heartfelt gratitude to the Eluna TC team for their work.
```

Comment:
let's extend that not only to a single emulator. There are great people working on various other emus. I wouldn't wanna feel them left out.